src/loader.c

```diff
@@ -168,11 +192,16 @@ static void *imv_loader_bg_new_img(void *data)
 
   int num_frames = 1;
   FIMULTIBITMAP *mbmp = NULL;
-  FIBITMAP *bmp = NULL;
   int width, height;
   int raw_frame_time = 100; /* default to 100 */
 
   if(fmt == FIF_GIF) {
+    if(load_from_memory == 1){
```

Comment:
We can support gifs: `FreeImage_LoadMultiBitmapFromMemory()`. Just have to do the if memory/path check and use the appropriate function.